app/lib/matador/components/job-info/JobInfoPanel.tsx

```diff
@@ -67,7 +67,7 @@ export const JobInfoPanel = ({
             </Grid.Col>
           )}
 
-          {attemptsMade && (
+          {attemptsMade !== undefined && (
```

Comment:
Why this check?

If an object is not undefined or null, It is casted as `true`

Reply:
If you do that, given that attemptsMade is a number, that statement return the value of attemptsMade and the React Component

Reply:
Oh, right, just a consideration:

use `null` instead of `undefined`.

Reply:
Use `!=`, not `!==`